pyiron/atomistics/structure/atoms.py

```diff
@@ -192,6 +193,7 @@ def __init__(
         else:
             self.dimension = 0
         self.visualize = Visualize(self)
```

Comment:
Should not self.visualize and self.analyse be hidden variables plus the corresponding properties to prevent the user from modifying them accidentally.

Reply:
The ultimate goal is to reduce the number of items in `Atoms`, since the tab completion list is now a total chaos. So for this one, no, it shouldn't be a hidden variable.

For `self.visualize` I'm less sure though.

Reply:
I mean:
```
def __init__(self):
    self._visualize = ...

@property
def visualize(self):
    return self._visualize 
```

Reply:
Ah sounds like a good idea